modules/@apostrophecms/schema/ui/apos/components/AposInputCheckboxes.vue

```diff
@@ -49,12 +58,12 @@ export default {
 
       if (this.field.min) {
         if ((values != null) && (values.length < this.field.min)) {
-          return 'min';
+          return `min ${this.field.min}`;
```

Comment:
does this break localization?

Reply:
No but the error message was giving no information about the min value which is weird and unclear for user. Works for all fields.

Reply:
Doesn't this show the word "min" or "max" unlocalized though?

Reply:
Oh yes actually it does